assistant_dists/dream_voice/docker-compose.override.yml

```diff
@@ -0,0 +1,223 @@
+services:
+  agent:
+    command: sh -c 'bin/wait && python -m deeppavlov_agent.run agent.channel=telegram agent.telegram_token=$TG_TOKEN agent.pipeline_config=assistant_dists/dream_voice/pipeline_conf.json'
+    environment:
+      WAIT_HOSTS: "dff-program-y-skill:8008, sentseg:8011, convers-evaluation-selector:8009, 
+          dff-intent-responder-skill:8012, intent-catcher:8014, badlisted-words:8018,
```

Comment:
badlisted -- you wanted to remove it

Reply:
![image](https://github.com/deeppavlov/dream/assets/92857699/c4a393da-d65d-472e-a552-becf0c845ad4)

maybe forgot to `git add`, check again please